### PR TITLE
app: Set `theme.tooltipTheme.preferBelow` to false

### PR DESCRIPTION
**Pull request by chrisbobbe**

### CODE REVIEW

**Comment by chrisbobbe** [comment]
To see the effect of this, try long-pressing the send button in the compose box, and see if your finger is obscuring the tooltip that appears.

File: lib/widgets/app.dart [hunk 1]
```diff
@@ -22,7 +22,12 @@ class ZulipApp extends StatelessWidget {
       //   https://api.flutter.dev/flutter/material/ColorScheme/ColorScheme.fromSeed.html
       // Or try this tool to see the whole palette:
       //   https://m3.material.io/theme-builder#/custom
-      colorScheme: ColorScheme.fromSeed(seedColor: kZulipBrandColor));
+      colorScheme: ColorScheme.fromSeed(seedColor: kZulipBrandColor),
+      // `preferBelow: false` seems like a better default for mobile;
+      // the area below a long-press target seems more likely to be hidden by
+      // a finger or thumb than the area above.
```
Comment (gnprice) [inline lib/widgets/app.dart:28]: Hmm, yeah.

Is there an upstream issue for this? Seems like this would be the right default for Flutter to have overall.

**Review by gnprice — APPROVED** [review]
Thanks! Looks good; feel free to merge.

It seems like this would be a better default for Flutter's tooltips to have in general. (Not only in mobile but on desktop too — the pointer on the screen is likely to extend down rather than up.) So I'd encourage you to find or write an issue upstream, and then send a PR to try to get it fixed there.

**Comment by chrisbobbe** [comment]
Sure:

- https://github.com/flutter/flutter/pull/133006
- https://github.com/flutter/flutter/pull/133007

**Pull request merged**